frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mention/mention_sub_page_bloc.dart

```diff
@@ -0,0 +1,112 @@
+import 'dart:async';
+
+import 'package:appflowy/plugins/trash/application/prelude.dart';
+import 'package:appflowy/workspace/application/view/prelude.dart';
+import 'package:appflowy_backend/protobuf/flowy-folder/protobuf.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'mention_sub_page_bloc.freezed.dart';
+
+class MentionSubPageBloc
```

Comment:
This bloc appears to have the same functionalities as the `mention_page_bloc`.

Reply:
I myself to prefer to keep the business logic separate for inline page references and inline child pages.